src/Account/TSPreKeyManager.m

```diff
@@ -383,6 +390,7 @@ + (void)clearSignedPreKeyRecordsWithKeyId:(NSNumber *)keyId success:(void (^_Nul
                 [dateFormatter stringFromDate:signedPrekey.generatedAt],
                 signedPrekey.wasAcceptedByService);
 
+            oldSignedPreKeyCount--;
```

Comment:
We also discussed using a different (longer) `kSignedPreKeysDeletionTime` for signed prekeys that were generated before February. Should we do that too?